docker/README.md

```diff
@@ -0,0 +1,13 @@
+Simple template for a Docker Compose setup with a SPARQL Endpoint and RDF browser for a small knowledge base.
```

Comment:
* probably worth saying it's Virtuoso-based
* no need for "small" qualifier, as the Virtuoso Open Source engine in this template can handle (at least) 60 Billion triples (see https://sparql.uniprot.org/ for a current deployment)
```suggestion
Simple template for a Docker Compose setup based on [Virtuoso](https://github.com/openlink/virtuoso-opensource/), providing an RDF store, SPARQL Endpoint, and RDF browser (among other available features) for a quick knowledge base deployment.
```

Reply:
The "small" qualifier was added because the RickView RDF browser uses a separate in-memory graph from the Sophia Rust RDF library. But because I will switch the RDF browser for another one, the qualifier can be removed.